planet/src/main/java/com/google/planet/servlets/ItineraryServlet.java

```diff
@@ -56,7 +58,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
             try {
                 ItineraryGenerator itineraryGenerator = new ItineraryGenerator();
-                List<ItineraryItem> itinerary = itineraryGenerator.generateItinerary(events);
+                List<ItineraryItem> itinerary = new ArrayList();
+                if (optimizedString.equals("true")){
+                    itinerary = itineraryGenerator.generateItinerary(events, ItineraryOrder.OPTIMIZED);
+                } else if (optimizedString.equals("false")){
+                    itinerary = itineraryGenerator.generateItinerary(events, ItineraryOrder.UNOPTIMIZED);
+                } else {
+                    throw new ItineraryException("Optimized can only be true or false");
```

Comment:
Technically it would never be anything other than true/false unless the user changes something in the dev tools console. And if that's the case I think it would be okay to skip the default